content/airrohr/nl/02-assemble.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Montage
+---
+
+> ⚠️ **BELANGRIJK**
+Installeer eerst de firmware voordat je de sensor monteert
+See __firmware flasher__ section.
+
+### NodeMCU v3
+Let op: onze instructies gaan over versie 3 van de NodeMCU. Deze kan je herkennen aan de pinnen VU en G (zie tekening).
+
+<img src="../docs/airrohr-wiring-sds011-bme280.jpg" style="width:40%; margin-top: 3em"/>
+<small>Copyright: roman-minyaylov, MIT License</small>
+
+
+<img src="../docs/nodemcu-v3-bme280.jpeg" style="margin-top: 1em"/>
```

Comment:
The path is not correct, here is the correct one:
`<img src="../docs/airrohr/airrohr-wiring-sds011-bme280.jpg" style="width:40%; margin-top: 3em;display: block"/>`